### PR TITLE
Remove intermediary last cover.

### PR DESCRIPTION
**Pull request by toots**
@vitoyucepi is there any reason we're keeping the last two covers in this function?

### CODE REVIEW

**Comment by vitoyucepi** [comment]
I don't know the original PR has only one cover.
Ref: ef4d2bac3adad72ed563b65362536f90e53bf71c, #3651.

**Comment by toots** [comment]
> I don't know the original PR has only one cover. Ref: [ef4d2ba](https://github.com/savonet/liquidsoap/commit/ef4d2bac3adad72ed563b65362536f90e53bf71c), #3651.

That might have been me working against this bug. Duh of course I didn't add a commit comment.

**Review by vitoyucepi — APPROVED** [review]

File: src/lang/builtins_string.ml [hunk 1]
```diff
@@ -29,6 +29,15 @@ let _ =
...
 
+let _ =
+  Lang.add_builtin ~base:string "digest" ~category:`String
+    ~descr:"Return an MD5 digest for the given string."
+    [("", Lang.string_t, None, None)]
+    Lang.string_t
+    (fun p ->
+      let data = Lang.to_string (List.assoc "" p) in
+      Lang.string Digest.(to_hex (string data)))
+
```
Comment (vitoyucepi) [inline src/lang/builtins_string.ml:40]: Why is this change related to the cover?

Comment (toots) [reply src/lang/builtins_string.ml:40]: Not at all. I tried something, realized we didn't have `string.digest` and the reverted the changes but kept the new function.

I was trying to store the cover with a digest of its content but then realized we also need to remove them so I reverted..

Comment (toots) [reply src/lang/builtins_string.ml:40]: I've been testing it and reviewed your remark about the temp file, you are correct, the file should always be either the default cover or a unique temp file.

Comment (toots) [reply src/lang/builtins_string.ml:40]: I'll add this directly to main and will rebase here.

**Comment by vitoyucepi** [comment]
After merging this PR, I'd like to reopen #4005 to continue the investigation since you're rolling back fixes from a4cb8e2bc8f05b4b56d95e234ef7e607db77524c.
